lib/vector-buffers/proptest-regressions/topology/acks.txt

```diff
@@ -2,6 +2,6 @@
 # automatically read and these particular cases re-run before any
 # novel cases are generated.
 #
-# It is recommended to check this file in to source control so that
+# It is recommended to check this file into source control so that
```

Comment:
`into` changes have been fairly controversial. I'm happy to drop them or anything else.

Reply:
I think it is good for me, but happy to get a second internal opinion.